### DISCUSSION TITLE
Ayuda con practica problema de Flexbox

### DISCUSSION THREAD

**Discussion by kerizrh** [Launch-X-Latam/MisionFrontEnd#358] [Q&A]
Hola a todos, estaba haciendo el CSS para mi practica de la pastelería pero hay algo que me esta dando inconvenientes.

Quiero tener 3 divs alineados dentro de un div contenedor, algo como esto:

[[LEFT]       [CENTER]        [RIGHT]]

Así que me puse salgo similiar a esto en mi HTML:

<div id="container" class="nombre de mi clase">

 <div id="left"  class="nombre de mi clase"></div>

 <div id="right" class="nombre de mi clase"></div>

 <div id="center" class="nombre de mi clase"></div>

</div>

Y en el CSS puse esto:

#container{width:100%;}
#left{float:left;width:100px;}
#right{float:right;width:100px;}
#center{margin:0 auto;width:100px;}

Pero se convierte en algo similar a esto:

[[LEFT]       [CENTER]              
                                    [RIGHT]

El ultimo div que sería "right" queda abajo y no logro alinearlo con los otros dos, no encuentro cómo solucionarlo, espero que me puedan ayudar. Gracias. 

![image](https://user-images.githubusercontent.com/114261500/200447669-5e63bd8a-cf2c-4ccc-8141-7a784ed2b56d.png)

**Comment by arieltruji** [comment]
Por lo que entendi de tu codigo fijate que cada div dentro del contenedor principal tenga un % acorde a como se van acomodando, en flexbox si utilizas el flex-wrap puedes organizarlos uno al lado del otro sin necesidad de utilizar float. Si tenes tu codigo corriendo en el navegador, pulsa F12 y prueba con la parte de estilos y display flex.

**Reply by kerizrh** [reply]
¡Muchas gracias! tu comentario me ayudo. 
Utilizando Flex-wrap y corrigiendo unas partes del html logré corregirlo.

**Comment by Jramo5** [comment]
div.padre{
    display: flex;
    flex-direction: row; /* Alineara todo en filas de izquierda a derecha*/
    flex-wrap: wrap; /* Con esto rompera la fila si el ancho es menor a lo necesario para que un hijo se alineen al lado de otro*/
}

div.hijo{
    width:33.33%; /* Si tienes tres div hijos dividiras 100% / 3 ---- Esto hara que los tres se alineen uno a la par de otro */
}

**Reply by Jramo5** [reply]
espero te ayude de algo, no soy tan bueno 😅
